.golangci.yml

```diff
@@ -0,0 +1,21 @@
+---
+linters:
+  enable:
+  # Checks for non-ASCII identifiers
+  - asciicheck
+  # Computes and checks the cyclomatic complexity of functions.
+  - gocyclo
+  # Inspects source code for security problems.
+  - gosec
+
+linters-settings:
+  gocyclo:
+    # Minimal code complexity to report.
+    # https://github.com/fzipp/gocyclo#readme
+    min-complexity: 20
```

Comment:
Are we bumping the `min-complexity` from the default?

Reply:
Yes, we are. [The default is 30, but 10-20 is recommended](https://github.com/golangci/golangci-lint/blob/master/.golangci.reference.yml#L555-L558).

Reply:
I've pushed a commit to document that.